ledger-core/network/integration/network_test.go

```diff
@@ -43,15 +37,16 @@ func TestJoinerNodeConnect(t *testing.T) {
 }
 
 func TestNodeConnectInvalidVersion(t *testing.T) {
+	t.Skip("protocol version should not be exposed")
```

Comment:
is it for security purpose?

Reply:
not really, just a lazy writer :)
full text is "protocol version should not be exposed to app logic level - it has to be handled/mapped at protocol level and covered with unit tests then"